spki/src/algorithm.rs

```diff
@@ -16,7 +16,7 @@ use der::{
 /// ```
 ///
 /// [RFC 5280 Section 4.1.1.2]: https://tools.ietf.org/html/rfc5280#section-4.1.1.2
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
```

Comment:
This type already impls `ValueOrd`

Reply:
Dropped Ord and PartialOrd from AlgorithmIdentifier and SignerInfo. Stubbed out ValueOrd for SignerInfo (not needed now and CMS is well short of complete at present).